test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -275,7 +276,7 @@ func GetSMTLevel(cpuID int, node *corev1.Node) int {
 
 // GetNumaNodes returns the number of numa nodes and the associated cpus as list on the node
 func GetNumaNodes(node *corev1.Node) (map[int][]int, error) {
-	lscpuCmd := []string{"lscpu", "-e=cpu,node", "-J"}
+	lscpuCmd := []string{"lscpu", "-e=node,core,cpu", "-J"}
```

Comment:
we don't seem to need this new data in this function, so maybe omit this change?

Reply:
we do need i think because we need per numa node per core siblings which lscpu -e=node,core,cpu . In the GetNumaNodes we don't use the core fields but since GetnNumaNodes and GetCoreSiblings share the same Structure we are using the same command.